lib/active_utils/country.rb

```diff
@@ -313,6 +313,20 @@ def to_s
       { :alpha2 => 'AX', :name => 'Åland Islands', :alpha3 => 'ALA', :numeric => '248' }
     ]
 
+    # Does this country use postal codes?  Default is true.
+    COUNTRY_USES_POSTALCODE = %w(
+      QA BZ BS BF BJ AG AE AI AO AW HK
+      FJ ML JM ZW YE UG TV TT TG TD PA
+      CW
+    ).inject(Hash.new(true)) do |map, country|
```

Comment:
It would. One could argue that hash lookup is faster than `Array#include?` but this is such a small array. I've changed to your form.